.mailmap

```diff
@@ -0,0 +1,4 @@
+Camille Moulin <cmoulin@inno3.fr> <camillem@users.noreply.github.com>
+Piotr Nowakowski <piotr.nowakowski@meelogic.com>
```

Comment:
Why do we need this line when there is only one address?

Reply:
To fix the name in case of `piotr.nowakowski <piotr.nowakowski@meelogic.com>`.